assets/app/views/settings.html

```diff
@@ -72,24 +72,48 @@
       <h2>
         <span>Resource limits</span>
         <i style="margin-left: 5px; cursor: help;" class="small pficon pficon-help" data-toggle="tooltip" data-placement="right"
-          title="Resource limits define minimum and maximum values that can be created for each resource type."></i>
+          title="Resource limits define minimum and maximum values that can be requested for each resource type."></i>
```

Comment:
So upstream uses the term "compute resources" which I think is more accurate than generic "resources".

How about this?

```Resource limits define minimum and maximum constraints for compute resources.```